thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss

```diff
@@ -1,16 +1,35 @@
-.application-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
+.dashboard-container {
+  &__body {
+    background-color: #FFF;
+    margin-bottom: 24px;
+    padding: 0 15px;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+  }
+
+  &__title {
+    margin: 30px 0 15px 0;
 
-  &__dropdown {
-    width: 200px;
+    span {
+      margin-left: 5px;
+    }
+  }
+
+  &__application-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+
+    &-dropdown {
+      width: 200px;
+    }
   }
-}
 
-.anomalies-table {
-  &__column-cell {
+  &__redirect-link{
     display: flex;
-    align-items: center;
+    justify-content: center;
+    border: solid 1px #CFCFCF;
```

Comment:
Let's not hard code any colors. 

We should utilize those we have here: https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/wrapper/_color.scss

Reply:
DONE